[PEN-631] GitHub package config

shk172 opened the pull request:
don't merge until documentation is done

shk172 committed:
PEN-631 Add github registry config

shk172 committed:
PEN-631 Refactor arc-test-org name to wpmedia

JackHowa reviewed (commented):
Nice work. Any readme process changes?

shk172 committed:
Publish

 - @wpmedia/article-body-block@1.5.7
 - @wpmedia/article-tag-block@1.2.2
 - @wpmedia/author-bio-block@0.6.3
 - @wpmedia/author-content-source-block@0.1.4
 - @wpmedia/byline-block@0.3.3
 - @wpmedia/card-list-block@0.1.5
 - @wpmedia/collections-content-source-block@0.1.2
 - @wpmedia/content-api-source-block@0.1.3
 - @wpmedia/date-block@1.4.2
 - @wpmedia/default-output-block@1.2.25
 - @wpmedia/double-chain-block@0.1.1
 - @wpmedia/extra-large-manual-promo-block@0.0.9
 - @wpmedia/extra-large-promo-block@0.0.8
 - @wpmedia/footer-block@1.6.5
 - @wpmedia/full-author-bio-block@0.2.5
 - @wpmedia/gallery-block@0.2.6
 - @wpmedia/header-nav-block@1.4.4
 - @wpmedia/headline-block@1.3.2
 - @wpmedia/htmlbox-block@0.2.2
 - @wpmedia/large-manual-promo-block@0.0.9
 - @wpmedia/large-promo-block@0.0.8
 - @wpmedia/lead-art-block@1.5.6
 - @wpmedia/links-bar-block@1.2.2
 - @wpmedia/medium-manual-promo-block@0.0.9
 - @wpmedia/medium-promo-block@0.0.8
 - @wpmedia/numbered-list-block@2.0.1
 - @wpmedia/overline-block@0.3.2
 - @wpmedia/results-list-block@1.4.1
 - @wpmedia/right-rail-block@0.1.11
 - @wpmedia/section-title-block@0.1.3
 - @wpmedia/share-bar-block@1.2.3
 - @wpmedia/shared-styles@0.0.5
 - @wpmedia/simple-list-block@1.0.2
 - @wpmedia/simple-results-list-block@0.0.8
 - @wpmedia/single-chain-block@0.2.2
 - @wpmedia/site-hierarchy-content-block@2.0.1
 - @wpmedia/small-manual-promo-block@0.0.9
 - @wpmedia/small-promo-block@0.0.8
 - @wpmedia/story-feed-author-content-source-block@0.1.3
 - @wpmedia/story-feed-query-content-source-block@0.1.3
 - @wpmedia/story-feed-sections-content-source-block@0.1.3
 - @wpmedia/story-feed-tag-content-source-block@0.1.2
 - @wpmedia/subheadline-block@1.3.2
 - @wpmedia/tag-title-block@0.2.3
 - @wpmedia/tags-content-source-block@0.0.7
 - @wpmedia/top-table-list-block@0.1.3
 - @wpmedia/triple-chain-block@0.1.1
 - @wpmedia/unpublished-content-source-block@0.2.1
 - @wpmedia/video-player-block@0.1.2

shk172 committed:
Published all to github

shk172 committed:
Publish

 - @wpmedia/article-body-block@1.5.8
 - @wpmedia/article-tag-block@1.2.3
 - @wpmedia/author-bio-block@0.6.4
 - @wpmedia/author-content-source-block@0.1.5
 - @wpmedia/byline-block@0.3.4
 - @wpmedia/card-list-block@0.1.6
 - @wpmedia/collections-content-source-block@0.1.3
 - @wpmedia/content-api-source-block@0.1.4
 - @wpmedia/date-block@1.4.3
 - @wpmedia/default-output-block@1.2.26
 - @wpmedia/double-chain-block@0.1.2
 - @wpmedia/extra-large-manual-promo-block@0.0.10
 - @wpmedia/extra-large-promo-block@0.0.9
 - @wpmedia/footer-block@1.6.6
 - @wpmedia/full-author-bio-block@0.2.6
 - @wpmedia/gallery-block@0.2.7
 - @wpmedia/header-nav-block@1.4.5
 - @wpmedia/headline-block@1.3.3
 - @wpmedia/htmlbox-block@0.2.3
 - @wpmedia/large-manual-promo-block@0.0.10
 - @wpmedia/large-promo-block@0.0.9
 - @wpmedia/lead-art-block@1.5.7
 - @wpmedia/links-bar-block@1.2.3
 - @wpmedia/medium-manual-promo-block@0.0.10
 - @wpmedia/medium-promo-block@0.0.9
 - @wpmedia/numbered-list-block@2.0.2
 - @wpmedia/overline-block@0.3.3
 - @wpmedia/results-list-block@1.4.2
 - @wpmedia/right-rail-block@0.1.12
 - @wpmedia/section-title-block@0.1.4
 - @wpmedia/share-bar-block@1.2.4
 - @wpmedia/shared-styles@0.0.6
 - @wpmedia/simple-list-block@1.0.3
 - @wpmedia/simple-results-list-block@0.0.9
 - @wpmedia/single-chain-block@0.2.3
 - @wpmedia/site-hierarchy-content-block@2.0.2
 - @wpmedia/small-manual-promo-block@0.0.10
 - @wpmedia/small-promo-block@0.0.9
 - @wpmedia/story-feed-author-content-source-block@0.1.4
 - @wpmedia/story-feed-query-content-source-block@0.1.4
 - @wpmedia/story-feed-sections-content-source-block@0.1.4
 - @wpmedia/story-feed-tag-content-source-block@0.1.3
 - @wpmedia/subheadline-block@1.3.3
 - @wpmedia/tag-title-block@0.2.4
 - @wpmedia/tags-content-source-block@0.0.8
 - @wpmedia/top-table-list-block@0.1.4
 - @wpmedia/triple-chain-block@0.1.2
 - @wpmedia/unpublished-content-source-block@0.2.2
 - @wpmedia/video-player-block@0.1.3

shk172 committed:
refactor dependency versions

shk172 committed:
package updates

shk172 committed:
merge conflicts

shk172 committed:
newest lock files

shk172 committed:
update header block

shk172 committed:
failed publish

shk172 committed:
temporarily removing test hooks for publishing

shk172 committed:
Publish

 - @wpmedia/article-body-block@1.5.10
 - @wpmedia/article-tag-block@1.2.5
 - @wpmedia/author-bio-block@0.6.6
 - @wpmedia/author-content-source-block@0.1.7
 - @wpmedia/byline-block@0.3.6
 - @wpmedia/card-list-block@0.1.9
 - @wpmedia/collections-content-source-block@0.1.5
 - @wpmedia/content-api-source-block@0.1.6
 - @wpmedia/date-block@1.4.5
 - @wpmedia/default-output-block@1.2.28
 - @wpmedia/double-chain-block@0.1.4
 - @wpmedia/extra-large-manual-promo-block@0.0.14
 - @wpmedia/extra-large-promo-block@0.0.13
 - @wpmedia/footer-block@1.6.8
 - @wpmedia/full-author-bio-block@0.2.8
 - @wpmedia/gallery-block@0.2.9
 - @wpmedia/header-block@0.2.2
 - @wpmedia/header-nav-block@1.4.7
 - @wpmedia/headline-block@1.3.5
 - @wpmedia/htmlbox-block@0.2.5
 - @wpmedia/large-manual-promo-block@0.0.14
 - @wpmedia/large-promo-block@0.0.13
 - @wpmedia/lead-art-block@1.5.9
 - @wpmedia/links-bar-block@1.2.5
 - @wpmedia/medium-manual-promo-block@0.0.15
 - @wpmedia/medium-promo-block@0.0.14
 - @wpmedia/numbered-list-block@2.0.4
 - @wpmedia/overline-block@0.3.5
 - @wpmedia/results-list-block@1.4.4
 - @wpmedia/right-rail-block@0.1.14
 - @wpmedia/section-title-block@0.1.6
 - @wpmedia/share-bar-block@1.2.6
 - @wpmedia/shared-styles@0.0.8
 - @wpmedia/simple-list-block@1.0.5
 - @wpmedia/simple-results-list-block@0.0.11
 - @wpmedia/single-chain-block@0.2.5
 - @wpmedia/site-hierarchy-content-block@2.0.5
 - @wpmedia/small-manual-promo-block@0.0.15
 - @wpmedia/small-promo-block@0.0.14
 - @wpmedia/story-feed-author-content-source-block@0.1.6
 - @wpmedia/story-feed-query-content-source-block@0.1.6
 - @wpmedia/story-feed-sections-content-source-block@0.1.7
 - @wpmedia/story-feed-tag-content-source-block@0.1.5
 - @wpmedia/subheadline-block@1.3.5
 - @wpmedia/tag-title-block@0.2.6
 - @wpmedia/tags-content-source-block@0.0.10
 - @wpmedia/top-table-list-block@0.1.9
 - @wpmedia/triple-chain-block@0.1.4
 - @wpmedia/unpublished-content-source-block@0.2.5
 - @wpmedia/video-player-block@0.1.5

shk172 committed:
putting test hooks back

shk172 committed:
latest lock files

shk172 committed:
temp remove tests again

shk172 committed:
Publish

 - @wpmedia/article-body-block@1.5.11
 - @wpmedia/card-list-block@0.1.10
 - @wpmedia/extra-large-manual-promo-block@0.0.15
 - @wpmedia/extra-large-promo-block@0.0.14
 - @wpmedia/header-block@0.2.3
 - @wpmedia/large-manual-promo-block@0.0.15
 - @wpmedia/large-promo-block@0.0.14
 - @wpmedia/lead-art-block@1.5.10
 - @wpmedia/medium-manual-promo-block@0.0.16
 - @wpmedia/medium-promo-block@0.0.15
 - @wpmedia/results-list-block@1.4.5
 - @wpmedia/simple-list-block@1.0.6
 - @wpmedia/small-manual-promo-block@0.0.16
 - @wpmedia/small-promo-block@0.0.15
 - @wpmedia/top-table-list-block@0.1.10

shk172 committed:
put tests back

brentmiller1973 reviewed (approved):
Bet that was fun!  :-P

JackHowa reviewed (commented):
I think the simple results list shouldn't be in here. One more thing is in the .eslintrc there's a couple ignores that may have to be updated to ignore wp media

shk172 committed:
Adding documentation for setting up github packages, deleting simple results list block

shk172 committed:
Adding documentation for setting up github packages, deleting simple results list block

shk172 committed:
Merge branch 'master' into PEN-631-GithubPackageConfig
